utils/discord-actions.js

```diff
@@ -26,7 +29,39 @@ const generateDiscordProfileImageUrl = async (discordId) => {
   }
 };
 
+const generateDiscordInviteLink = async () => {
+  try {
+    const channelId = config.get("discordNewComersChannelId");
+    const authToken = jwt.sign({}, RDS_SERVERLESS_PRIVATE_KEY, {
+      algorithm: "RS256",
+      expiresIn: RDS_SERVERLESS_TTL,
+    });
+
+    const inviteOptions = {
+      channelId: channelId,
```

Comment:
Just a question:  Isn't there a way where we can specify the times where we can use this link? Do we follow that here? 
Eg: if someone was kicked from the server can they use the same link to join again?

Reply:
This invitation link is designed for a single user to join, and once it has been used, it becomes invalid.